mkdocs.yml

```diff
@@ -73,6 +76,25 @@ plugins:
           Alternatives, Inspiration and Comparisons: Alternativas, Inspiración y Comparaciones
           API Reference: Referencia de la API
           Changelog: Registro de Cambios
+        tr:
+          Overview: Genel Bakış
+          Tutorial - User Guide: Kılavuz - Kullanıcı Kılavuzu
+          Getting Started: Başlarken
+          Configurations: Konfigürasyonlar
```

Comment:
TDK konfigürasyon için yapılandırma demiş ancak doğru bir karşılık olmamış bence. Burada con- eki var bu da Türkçemizdeki -(X)ş- eki ile bir sözcük kullanarak karşılığını vereceğimizi düşündürtür.

Biçimleştirim sözcüğü yapılandırma sözcüğünden daha doğru olur bence.

Reply:
"Biçimleşim" kelimesi, tahmin ediyorum ki günlük Türkçe'de aktif kullanılan bir kelime değil. Yapılandırma kullanımı daha uygun olabilir belki.